src/conversion/downcastdispatcher.js

```diff
@@ -440,7 +451,19 @@ export default class DowncastDispatcher {
 
 		const name = data.item.name || '$text';
 
-		this.fire( type + ':' + name, data, consumable, this.conversionApi );
+		this.conversionApi.consumable = consumable;
```

Comment:
I will be cleaner if you do this when you create consumable and do not pass `consumable` to `_testAndFire`.